doc/qbittorrent.1.md

```diff
@@ -3,34 +3,165 @@
 % January 16th 2010
 
 # NAME
-qBittorrent - a Bittorrent client written in C++ / Qt
 
+**qBittorrent** - a BitTorrent client written in C++ / Qt
 
 # SYNOPSIS
-**qbittorrent** `[--no-splash] [--webui-port=x] [TORRENT_FILE | URL]...`
-
-**qbittorrent** `--help`
-
-**qbittorrent** `--version`
 
+| **qbittorrent** \[_OPTIONS_] \[_TORRENT_FILE_ | _URL_ ...]
+| **qbittorrent** **-h | -v**
```

Comment:
I think `-h`, `-v` are already implied in the above `OPTIONS`, maybe we don't need this?

Reply:
No, it's a separate form: it you specify `-h` or `-v`, nothing else is allowed.